lib/util/edit.ts

```diff
@@ -12,6 +12,7 @@ import { MemoryFileService } from './MemoryFileService.js';
 import { findFileUsage } from './findFileUsage.js';
 import { parseFile } from './parseFile.js';
 import { Output } from './Output.js';
+import * as Export from './export.js';
```

Comment:
I've used namespaces to organise the code. This way you can do stuff like

```
Export.DeclarationWholeExport.isFileFound
```

Tell me what you think about this

Reply:
I’ve been thinking about this, but can we use `export { foo } from 'foo'` to keep the style consistent? Sorry for the confusion🙏

Reply:
Done!